File: 27 - 2/arrays/arrays.py
def static_array():
    numbers = [0] * 5
    
    for i in range(5):
        numbers[i] = i + 1
    
    return numbers

def dynamic_array():
    numbers = []
    for i in range(5):
        numbers.append(i + 1)
    
    return numbers

def main():
    print("Static Array:", static_array())
    print("Dynamic Aray:", dynamic_array())
    
if __name__ == "__main__":
    main()

File: 3-3/arrays/go/arrays.go
package main

import (
	"fmt"
)

func staticArray() {
	var numbers [5]int

	for i := 0; i < 5; i++ {
		numbers[i] = i + 1
	}

	fmt.Println("Static Array:", numbers)

}

func dynamicArray() {
	var numbers []int

	for i := 0; i < 5; i++ {
		numbers = append(numbers, i + 1)
	}

	fmt.Println("Dynamic Array:", numbers)
}

func main() {
	staticArray()
	dynamicArray()
}